clang/utils/TableGen/ClangOpenCLBuiltinEmitter.cpp

```diff
@@ -83,24 +83,32 @@ struct BuiltinTableEntries {
 
 class BuiltinNameEmitter {
 public:
-  BuiltinNameEmitter(RecordKeeper &Records, raw_ostream &OS)
-      : Records(Records), OS(OS) {}
+  BuiltinNameEmitter(RecordKeeper &Records, raw_ostream &OS,
+                     llvm::StringRef Family)
+      : Records(Records), OS(OS), Family(Family),
+        ClassName((Family + "Builtin").str()) {}
 
   // Entrypoint to generate the functions and structures for checking
-  // whether a function is an OpenCL builtin function.
+  // whether a function is a builtin function.
   void Emit();
 
 private:
   // A list of indices into the builtin function table.
   using BuiltinIndexListTy = SmallVector<unsigned, 11>;
 
-  // Contains OpenCL builtin functions and related information, stored as
+  // Contains builtin functions and related information, stored as
   // Record instances. They are coming from the associated TableGen file.
   RecordKeeper &Records;
 
   // The output file.
   raw_ostream &OS;
 
+  // Family for which the builtin are for.
+  llvm::StringRef Family;
+
+  // Family for which the builtin are for.
```

Comment:
Looks like a copy of the comment at line 106.
```suggestion
  // Class for which the builtin are for.
```
?